tests/onnx/quantization/test_calculation_quantizer_params.py

```diff
@@ -0,0 +1,181 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import pytest
+from nncf.common.quantization.structs import QuantizationMode
+from nncf.common.quantization.structs import QuantizerConfig
+from nncf.experimental.onnx.quantization.quantizer_parameters import calculate_scale_zero_point
+from nncf.experimental.onnx.quantization.quantizer_parameters import calculate_activation_quantizer_parameters
+from nncf.experimental.onnx.quantization.quantizer_parameters import calculate_weight_quantizer_parameters
+from nncf.experimental.onnx.quantization.quantizer_parameters import get_level_low_level_high
+from nncf.experimental.onnx.quantization.quantizer_parameters import ONNXQuantizerLayerParameters
+from nncf.experimental.onnx.statistics.statistics import ONNXMinMaxTensorStatistic
+import numpy as np
+from dataclasses import dataclass
```

Comment:
I think it's more readable 
```suggestion
import pytest
import numpy as np
from dataclasses import dataclass

from nncf.common.quantization.structs import QuantizationMode
from nncf.common.quantization.structs import QuantizerConfig
from nncf.experimental.onnx.quantization.quantizer_parameters import calculate_scale_zero_point
from nncf.experimental.onnx.quantization.quantizer_parameters import calculate_activation_quantizer_parameters
from nncf.experimental.onnx.quantization.quantizer_parameters import calculate_weight_quantizer_parameters
from nncf.experimental.onnx.quantization.quantizer_parameters import get_level_low_level_high
from nncf.experimental.onnx.quantization.quantizer_parameters import ONNXQuantizerLayerParameters
from nncf.experimental.onnx.statistics.statistics import ONNXMinMaxTensorStatistic
```

Reply:
Done